apps/portals/src/configurations/elportal/synapseConfigs/cohortbuilder.ts

```diff
@@ -79,6 +79,11 @@ export const filesView: SynapseConfig = {
     isRowSelectionVisible: true,
     rowSelectionPrimaryKey: ['fileId'],
     additionalFiltersLocalStorageKey: 'cohort-builder-files-perspective',
+    combineRangeFacetConfig: {
```

Comment:
Looks like they added the minAge and maxAge column to the file perspective of the Cohort Builder!